### DISCUSSION TITLE
Where can we track releases and change logs?

### DISCUSSION THREAD

**Discussion by yannxaver** [resend/react-email#973] [General]
https://github.com/resendlabs/react-email/releases 

Last release is marked as April 25 but the package on npm has been updated after that. 

https://www.npmjs.com/package/react-email.

**Comment by Joeao** [comment]
The change log on the website is also quite dated https://react.email/docs/changelog

**Comment by thesaltygerman** [comment]
I jus use https://jsx.email now. react-email not good care.
